docs/devctr-image.md

```diff
@@ -16,6 +16,7 @@ registry. The Firecracker CI suite must also be updated to use the new image.
    you've ever built Firecracker from source.
 1. Access to both an `x86_64` and `aarch64` machines to build the container
    images.
+1. Ensure `aws --version` is >=2.0.0.
```

Comment:
I think maybe this happens with very old versions of v1. My awscli v1 does have it:

```sh
% aws --version
aws-cli/1.25.9 Python/3.8.13 Linux/5.4... botocore/1.27.9

% aws ecr get-login-password
```

Actually it's in the [docs](https://docs.aws.amazon.com/cli/latest/reference/ecr/get-login-password.html)

> This command is supported using the latest version of AWS CLI version 2 or in v1.17.10 or later of AWS CLI version 1.

Reply:
Updated to be `>=1.17.10`

Reply:
I just tried this on 1.18 (which is what comes preinstalled with Amazon Linux on my c5.metal instance) and it did not find the command `ecr-public` (the docs you link are for the `ecr` command). The docs do not say anything about `ecr-public get-login-password`.